README.md

```diff
@@ -7,7 +7,7 @@
 ChaosBot is a social coding experiment to see what happens when the absolute
 direction of a software project is turned over to the open source community.
 
-![There was clearly a kitty missing.](http://thecatapi.com/api/images/get?format=src&type=png&size=small)
+![Image of a kitten with overlay: "I WONDER IF BEING SANE MEANS DISREGARDING THE CHAOS THAT IS LIFE...PRETENDING ONLY AN INFINITESIMAL SEGMENT OF IT IS REALITY" Source: http://existentialpets.tumblr.com/post/48069725344](assets/chaos-kitty.jpg)
```

Comment:
Probably source should be made clearner, perhaps as a link? Because currently it's an alt tag, which most users won't see.

That said, it can be fixed after this is merged, as it's not important enough, and changes cause vote count to reset.